.gitignore

```diff
@@ -16,3 +16,4 @@ data.db
 # .vscode - We intentionally commit .vscode in order to share project-specific settings
 
 
+.idea
```

Comment:
I think this is a good idea - will make the lives of goland developers better.

Reply:
👍🏽 - I'm looking forward to the nix dev environment blog post from you folks.